core/src/datayoga_core/blocks/redis/utils.py

```diff
@@ -36,23 +41,26 @@ def get_client(connection_details: Dict[str, Any]) -> Redis:
     cacert = connection_details.get("cacert")
 
     try:
-        client = redis.Redis(
+        redis_client = Redis(
             host=host,
             port=port,
             username=connection_details.get("user"),
             password=connection_details.get("password"),
-            ssl=(key is not None and cert is not None) or cacert is not None,
+            ssl=cacert is not None or cert is not None or key is not None,
             ssl_keyfile=key,
             ssl_password=key_password,
             ssl_certfile=cert,
-            ssl_cert_reqs=ssl.CERT_REQUIRED if cacert else ssl.CERT_NONE,
             ssl_ca_certs=cacert,
+            ssl_ciphers=SSL_CIPHERS,  # Customize TLS ciphers (Python 3.10+)
             decode_responses=True,
-            client_name="datayoga"
+            client_name="datayoga",
+            socket_timeout=connection_details.get("socket_timeout", 10.0),
+            socket_connect_timeout=connection_details.get("socket_connect_timeout", 2.0),
```

Comment:
are these the defaults we use today?

Reply:
We use those in RDI. We haven't used these new properties at all up until now.